config.json

```diff
@@ -72,6 +72,10 @@
         "coingecko-quote-currency": "usd",
         "worker-min-pool-liquidity-cap": 1
     },
+    "passthrough":{
```

Comment:
There are some config in `README.md`, shall we update it too? Maintaining same config at multiple places is quite repetitive task, maybe we should omit it from `README.md` and link to actual file instead?